PowerEditor/src/MISC/PluginsManager/Notepad_plus_msgs.h

```diff
@@ -35,7 +35,7 @@
 enum LangType {L_TEXT, L_PHP , L_C, L_CPP, L_CS, L_OBJC, L_JAVA, L_RC,\
 			   L_HTML, L_XML, L_MAKEFILE, L_PASCAL, L_BATCH, L_INI, L_ASCII, L_USER,\
 			   L_ASP, L_SQL, L_VB, L_JS, L_CSS, L_PERL, L_PYTHON, L_LUA,\
-			   L_TEX, L_FORTRAN, L_BASH, L_FLASH, L_NSIS, L_TCL, L_LISP, L_SCHEME,\
+			   L_TEX, L_FORTRAN, L_FORTRAN_77, L_BASH, L_FLASH, L_NSIS, L_TCL, L_LISP, L_SCHEME,\

```

Comment:
I may be incorrect, but I believe you'd want to put `L_FORTRAN_77` right before `L_EXTERNAL` because this shifts all the enum values...which can cause havoc on plugins that use these, else all of them would have to be re-compiled.